src/framework/preferences/css/stylus/utils/Fonts.styl

```diff
@@ -27,5 +60,21 @@ build-fonts-Enactors(textFont) {
                 font-family: unquote(font) !important;
             }
         }
-    }
+    }    
+
+    // Build any necessary @font-face definitions
+    for fontFamilyName in textFontFaces {
```

Comment:
Things seem to be working, so I'm not sure if this really matters, but as set up, the `@font-face` declaration comes after using the fonts in the CSS file.

Reply:
It's probably more intuitive to have it come before - I'll make the change.